src/Aspire.Hosting/Publishing/ManifestPublishingContext.cs

```diff
@@ -52,7 +52,9 @@ public sealed class ManifestPublishingContext(DistributedApplicationExecutionCon
 
         var fullyQualifiedManifestPath = Path.GetFullPath(ManifestPath);
         var manifestDirectory = Path.GetDirectoryName(fullyQualifiedManifestPath) ?? throw new DistributedApplicationException("Could not get directory name of output path");
-        var relativePath = Path.GetRelativePath(manifestDirectory, path);
+
+        var normalizedPath = path.Replace('\\', Path.DirectorySeparatorChar).Replace('/', Path.DirectorySeparatorChar);
```

Comment:
This isn't great on Unix since you can actually have backslashes in directory names.

Reply:
This was taken from the existing `PathNormalizer` utility class in `Aspire.Hosting`

Reply:
Maybe it should be fixed there too?

I know MSBuild doesn't care about path handling since you need to use slashes in in the .csproj, but I'm not sure how far we want to propagate that through the .NET stack. It feels "Windows-centric" to me.

Reply:
Logged #2788 to follow up

Reply:
Thanks!